### ISSUE TITLE
OwnedDrawData isn't actually Send

### ISSUE THREAD

**Issue by PJB3005** [imgui-rs/imgui-rs#815]
https://github.com/imgui-rs/imgui-rs/blob/844533445087e0aa18247701002d08df446fbffb/imgui/src/render/draw_data.rs#L314-L320

According to the description, it "can be used to retain draw data for use a different thread". Except... It's not `Send`? You can't actually safely use this on another thread.

**Comment by sanbox-irl** [comment]
I agree, this comment in the code is somewhat bizarre. I'm not really sure what the purpose of the type even is. How are you using it?

**Comment by PJB3005** [comment]
> I agree, this comment in the code is somewhat bizarre. I'm not really sure what the purpose of the type even is. How are you using it?

The implementation in `bevy_mod_imgui` relies on manually using `unsafe impl Send` to wrap it. Before my PR (that is linked) the scope of said `impl` was broadly unsound, now it's only to wrap this specific type.
